composer.json

```diff
@@ -45,18 +46,22 @@
         "phpstan/phpstan-strict-rules": "^0.12.0",
         "phpstan/phpstan-webmozart-assert": "0.12.12",
         "phpunit/phpunit": "^9.5",
-        "sylius-labs/coding-standard": "^4.0",
+        "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "symfony/browser-kit": "^4.4 || ^5.2",
         "symfony/debug-bundle": "^4.4 || ^5.2",
         "symfony/dotenv": "^4.4 || ^5.2",
         "symfony/intl": "^4.4 || ^5.2",
         "symfony/web-profiler-bundle": "^4.4 || ^5.2",
         "symplify/console-color-diff": "^8.0 || ^9.3",
-        "vimeo/psalm": "4.6.4",
-        "polishsymfonycommunity/symfony-mocker-container": "^1.0"
+        "vimeo/psalm": "4.24"
```

Comment:
Why this change?

Reply:
"vimeo/psalm": "4.6.4" was too low version for "bitbag/coding-standard" so I updated it to the newest.